reports/github.go

```diff
@@ -0,0 +1,64 @@
+package reports
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	ghinstallation "github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/google/go-github/v57/github"
+	"golang.org/x/exp/slog"
+)
+
+var (
+	ghAppName                 = os.Getenv("CICD_ROBOT_GITHUB_APP_NAME")
+	ghAppPrivateKey           = os.Getenv("CICD_ROBOT_GITHUB_APP_PRIVATE_KEY")
```

Comment:
coordinate with @ALRubinger to get `CICD_ROBOT_*` secrets available in this repo.